spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -44,6 +44,70 @@
 
       delete :logout, params: { SAMLRequest: 'foo' }
     end
+
+    let(:service_provider) do
+      create(:service_provider,
+             cert: nil, # override singular cert
+             certs: ['saml_test_sp'],
```

Comment:
**Question:** Is it worth adding a second cert to validate that it picks the right one? I can go either way.

Reply:
Sure, will add another one

Reply:
Updated in 08ccb099413dd218e5e80c1d78f3b75fe51526c2